Makefile

```diff
@@ -167,6 +167,36 @@ sumo_tarball_latest: checkout_leapdeps_develop pull_leapdeps setup_without_names
 pyinst:
 	pyinstaller -y pkg/pyinst/bitmask.spec
 
+pyinst-hacks:
+	mkdir -p dist/bitmask/cryptography/hazmat/primitives/src/
+	cp ~/src/cryptography-0.9.3/src/cryptography/hazmat/primitives/src/* dist/bitmask/cryptography/hazmat/primitives/src/
```

Comment:
This looks really hacky to me. `~/src/cryptography-0.9.3`? I guess is a left over from some iterations. Can it be removed?